README.md

```diff
@@ -30,6 +33,44 @@ but expect them to take a longer time until someone finds time to fix them.
 If you want to help improve the schemas, have a look at the [development
 documentation](CONTRIBUTING.md).
 
+## Jinja2 implicit vs explicit templating
+
+While Ansible might allow you to combine implicit and explicit templating, our
+schema will not. Our schemas will only allow you to use the recommended form,
+either by forbiding you to use the curly braces on implicit ones or forcing
+you to add them on explicit ones.
+
+Examples:
+
+```yaml
+- name: some task
+  command: echo 123
+  register: result
+  no_log: result.rc == 1 # <-- schema will not allow this
```

Comment:
that would be interpreted by ansible as a string, not a boolean, while `no_log: false` is still valid

Reply:
Yes. that is while I listed this as not accepted by the schema. `false` will work just fine, as it is a boolean, not a string.